### ISSUE TITLE
Other useful comments from our editors on the guidelines - comments welcome!

### ISSUE THREAD

**Issue by lyubomirpenev** [plazi/arcadia-project#30]
- [ ] Congratulations, this is a very good idea! 
Uniform citation is something I always promoted, but many authors find hard to follow. 
Just a few questions, suggestions:
1) In some cases I prefer a table with specimen data to the standard textual treatment, especially in papers that are not strictly taxonomic. I guess that this is still a good alternative? Should it be mentioned? 
2) Degree symbol: I suggest to add the ISO/ASCII codes for the correct degree symbol, I have seen many incorrect variants in mss that I edited or reviewed. 
Also male/female symbol still often give problems, a note on those in the guidelines might be helpful. 

- [ ] I have checked the new material guidelines. Most of it is clear and fine, however, the examples are typical zoological data, and I am very much missing the most critical data item for botany: collector+number information. According to the guidelines the collector should be listed including leg. behind the name; for botanical records the "leg." this should be replaced by the collection number, and if no collection number was issued by "s.n."; to any botanist it will be clear that whatever is listed before the number or the s.n. is the collector. But the collection number (which professional botanist will use) is critical for matching duplicates in other herbaria and is the main key used in botany to find specimens in databases etc.

So please make sure the data standard for botany is changed to include collector+number sequence in stead of the "leg.". Moreover, the specimens [e.g. 1 ♂, size] should be dropped for botany; only few plants have a single sex, and plants are usually sampled, not collected as a whole; there is no clear 1-1 relation between biological specimens and herbarium specimens, and a collection can consist of only a part of- to many actual plants. In standard botanical citations this actually is the place where one would use the collector+number reference. See screenshot from a recent paper by myself.

image.png
So I urge you to adapt these standards, for else I am sure there will be a lot of issues with botanical authors, and I would not be able to defend a policy not including collection numbers.

**Comment by myrmoteras** [comment]
keep collecting them. 

the issues with specimen, collection code, persistent identifier is something we need to discuss together with GBIF at least, because this is causing troubles.

Can you ask the author of the comments whether he has guidelines for the botanists to which we could refer?

**Comment by myrmoteras** [comment]
regarding the table:  I am for it. The question right now is how to deal with it to assure the data therein is linked to the respective treatment.

Are there examples for it?

**Comment by lyubomirpenev** [comment]
We do at this Plazi GitHub account.
The do not have specific guidelines but now I see a chance to adapt these in PhytoKeys. We are requesting feedback from our botanical and mycological editors.


--
Sent from Outlook Email App for Android вторник, 19 март 2019, 01:26PM +02:00 from Donat Agosti  notifications@github.com :

**Comment by lyubomirpenev** [comment]
On the long run yes, but I suggest we do these steps one by one starting with the most difficult - the in-text materials citations.

--
Sent from Outlook Email App for Android вторник, 19 март 2019, 01:27PM +02:00 from Donat Agosti  notifications@github.com :

**Comment by lyubomirpenev** [comment]
I add for record, comments and discussion below about using "collection number" in botany (do not mix it with Collection ID). "Collection number" is the collector name + a number of a certain gathering (e.g. leaves, flowers and pieces of wood from a single tree) that can be preserved on different herbarium sheets in different herbaria (but all sheets bear the same collection number which can link these with one another. Hence the question  to @gsautter: 

Can we recommend for botany publications replacement of the "Collector name + leg" as it stands now (typical for zoology) with "Collection number"? Collection number is a widely used term and identifies the collector name + gathering number. Will this somehow mix up your export to GBIF? Below is the proposed amendment:

"For botanical and mycological data, please use "Collector number" instead of "Collector name + Leg.". The collector's number encompasses all gatherings from a single specimen (e.g., leaves, flowers, piece of wood) which may be preserved on different herbarium sheets and in different herbaria."

- [ ] Comment 1: I don't mean specimen ID's. Unlike most zoological collections, it is in botany quite normal one gathering consists of several specimens. When I collect a tree, I cut 10 branches, which I conserve in 10 different newspapers, and I put some flowers in a spirit bottle and cut a piece of wood from its trunk. All these samples receive "my" collection number 6543. Of these samples, my home herbarium keeps 2 newpapers which become the herbarium sheets and the pot with pickled flowers and the wood sample. The other 8 newspapers are send to 8 other herbaria. All materials will be labelled "Wieringa 6543", which makes these collections traceable as one gathering. If a revision cites Wieringa 6543 (K,P) (so the author saw the Paris and Kew sheets), I know I can annotate my own WAG-sheets as well, since they are part of the same gathering.

Our own samples will each get their own barcode, so
Herbarium sheet 1: WAG0012345
Herbarium sheet 2: WAG0012346
Wood sample: WAG0012347
Spirit sample: WAG0012348
Herbarium sheet (at Paris): P01234567

So, this one gathering will have several barcodes attached, but since they are all part of the same gathering, they need to be cited as a single gathering (it is only one record in time and place of this species, it only now consist of parts).
The "gathering ID" is thus Wieringa 6543. The specimens have a unique barcode as an ID, and potentially the gathering per herbarium will have a unique record ID in their database, but this will be different for each database, and is usually not made visible or being shared, it is just a unique database identifier. Moreover, it may change when databases are imported into other (new) systems.

Since the gathering ID is so important in botany, and the collector name is part of gathering ID, it is senseless of repeating that as an explicit collector. In the cases the collector did not issue a number, we are used to adding "s.n." (sans numero) after the collector, which again makes it clear it is a collector name, therefore adding "leg." is not nessacery and is never done in botany (I am also an entomologist, and know it is a normal thing in zoology).

As an example, I propose:

GABON - Estuaire • J.J. Wieringa et al. 6543 (IEC, K, LBV, MO, P (P01234567), US, WAG (WAG0012345, WAG0012346, wood: WAG0012347, fl in spirit: WAG0012348)); 35 km NE of Libreville; 0°38.8'N, 9°58.9'E; alt. 30 m; 12 Feb. 2017; fl • E.L.A.N. Simons 1212 (P, LBV, WAG (WAG0987654), Z); Libreville, Sibang arboretum; 0°27.8'N 9°29.4'E; alt. 16 m; 15 Jul 2017; fr. - Ogooué-Maritime  • J.J. Bos 10123 (P, WAG); around Port Gentil; 0.82°S, 8.80°E; alt. 5 m; 4 Apr 1986; fl, fr.

Here I have added the phenology of the collections after the date. Might not be possible or relevant for all studies, but is nice if it can be cited as well, also as data from which the phenology of the species can be derived. In botany quite often also the habitat is described on labels, but it is not standard practice to copy these into the specimen citation; as is the often lengthy plant description. In specimen citation usually the additional collectors are not mentioned at all or they are replaced by "et al.". In case this is considered to be relevant they could be listed as well fully.
In this example I took some collections of which partly the barcodes were known to the researcher and I cited them in full, but for normal specimen citation this might not be needed. On the other hand: if one wants to link these specimens to the records already in GBIF the barcode is probably the only identifier with which this is easily done.

- [ ] I have a couple of questions. First, is there a commonly used and community accepted term for what you call "collection number"  or "gathering ID"? I am afraid that "collection number" would be easily mixed up with the DwC term "Collection ID". "Collector number" or "Collector's number" seem the most appropriate at first glance, I believe more than "Gathering ID". However it could be,  it is crucially important to avoid overlapping or misuse of terms, because materials citations will be a subject or automated conversion into XML and consequent indexing in GBIF.

Second, I wonder if it would be sufficient to add the following statement in the appropriate place in the guidelines:

For botanical and mycological data, please use "Collector number" [if we want to use this or another term?] instead of <Collector Name + leg>. The collector's number encompasses all gatherings from a single specimen (e.g., leaves, flowers, piece of wood) which may be preserved on different herbarium sheets and in different herbaria.

[Sample}

Naturally, we would need to discuss this with GBIF and the other PhytoKeys editors as we will really need a consensus on that in order to avoid problems with machine readability of the data.

**Comment by myrmoteras** [comment]
@lyubomirpenev please add this sort of issue to the Plazi communications issue tracker https://github.com/plazi/Plazi-Communications/issues

**Comment by gsautter** [comment]
I can surely add functionality tagging such collection numbers (for what I see in this ticket of the form `<lastName> <number>`). Would need a few example documents with such data in them for testing, of course.
Also, with last names involved, I have to wonder how affixes like "van" would be handled ... would it by "van Dyke 1234" or just "Dyke 1234"?

**Comment by lyubomirpenev** [comment]
Very good. We will put this amendment in the guidelines with respect to botanical data.

**Comment by myrmoteras** [comment]
Before we make changes, let's wait and see at least what GBIF will return.

As far as I can remember the issue of label transcription has been discussed and we decided with EJT that we add the label transcription in materialsCitation in-between, I believe with no access to our EJT guidelines right now, "".

I will check tomorrow morning
